Documentation/en-us/InstallingQuick.md

```diff
@@ -141,13 +141,81 @@ This is not "the one and only way" to use Carthage to manage dependencies.
 For further reference check out the [Carthage documentation](https://github.com/Carthage/Carthage/blob/master/README.md).
 
 ## [Swift Package Manager](https://github.com/apple/swift-package-manager)
-With the advent of the [swift.org](https://swift.org) open-source project, Swift now has an official, though nascent, package manager tool. Notably, this provides the possibility of using Quick on non-Apple platforms for the first time. Initial steps have been taken to allow using Quick to test projects using the Swift Package Manager, although frequent breakage is expected at this point since the tool is still under heavy development.
 
-Until further documentation has been written, the following repository may be useful as an example of how Quick can be declared as a dependency in a `Package.swift` file for SwiftPM:
+With the advent of the [swift.org](https://swift.org) open-source project, Swift now has an official package manager tool. Notably, this provides the possibility of using Quick on non-Apple platforms for the first time. You can use Swift Package Manager either by using it's integration with Xcode, or by editing your package's `Package.swift` file.
 
-https://github.com/Quick/QuickOnLinuxExample
+### Xcode Integration
 
-### (Not Recommended) Running Quick Specs on a Physical iOS Device
+To install Quick via XCode's Swift Package Manager integration, select your .xcodeproj file, then the project tab, then the Package Dependencies tab. Click on the "plus" button at the bottom of the list, then follow the wizard to add Quick to your project. Specify `https://github.com/Quick/Quick.git` as the url, and be sure to add Quick as a dependency of your unit test target, not your app target.
+
+### Package.swift
+
+Add Quick's github url as a dependency of your package, then as a dependency of your test target, like so:
+
+```swift
+// swift-tools-version:5.5
+
+import PackageDescription
+
+let package = Package(
+    name: "MyPackage",
+    products: [
+    	.library(name: "MyPackage", targets: ["MyPackage"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Quick/Quick.git", from: "5.0.1"),
+        .package(url: "https://github.com/Quick/Nimble.git", from: "9.2.1"),
+    ]
+    targets: [
+    	.target(name: "MyPackage", dependencies: []),
+    	.testTarget(name: "MyPackageTests", dependencies: ["Quick", "Nimble"])
+    ]
+)
+```
+
+#### Linux Support
+
+On Linux, you will need to also add a `LinuxMain.swift` at the root of the `Tests` subdirectory. The contents of this branches depending on the `swift-tools-version` your `Package.swift` file specifies.
+
+##### Swift Tools Version 5.4 and earlier
```

Comment:
Is it possible / easy for us to support only Swift 5.6 and above?

Supporting old Swift versions isn't as important as supporting old OS versions -- the language versions are compatible with all OSes essentially (and we're dropping Xcode 12 and below), and the Swift compiler has very good support for mixing Swift versions. So even if someone is using Swift 5.4 or 5.5 for some reason, this shouldn't have a huge impact on their project.

Reply:
As far as I can tell, the only way for us to support only 5.6 and above is to start using 5.6 features without guarding them with the appropriate `#if swift(>=5.6)`.